### PR TITLE
Change model.syllablePart

### LINKED ISSUE #968: Request for making several elements valid in mei-neume dev-5.0.0 schema
As shown in the figure below, the underlined elements are not valid in the mei-neume dev-5.0.0 schema. The details and examples are as follows. All the examples are from Einsiedeln (CH-E_611).

mei-Neumes   dev copy

1. `<accid>` is not valid both inside and outside a syllable, but it happens both inside and outside.

inside:
018 flat inside 1 - folio
018r flat inside 1 - mei

018r flat inside 2 - folio
018r flat inside 2 - mei

outside:
260r flat outside - folio
260r flat outside - mei

261v flat outside - folio
261v flat outside - mei

2. `<divLine>` is only valid inside a syllable, but it is mostly outside.

044r divisio outside - folio
044r divisio outside - mei

267v divisios outside - folio
267v divisios outside - mei

3. `<clef>`is only valid outside a syllable, but sometimes it is inside.

270v clef inside 1 and 2 - folio
270v clef inside 1 and 2 - mei

061v clef inside - folio
061v clef inside - mei

Compared to mei-all dev-5.0.0 schema, all the elements mentioned above but `<accid>` (also needs to be fixed) are valid both inside and outside a syllable. 

mei-all   dev copy

### PR DESCRIPTION
**Pull request by rettinghaus**
This PR removes `model.eventLike` from `model.syllablePart` and makes `clef` direct member of `model.syllablePart` instead. 
This solves the problem with `<custos>` being allowed in `<syllable>` from #968 saves us having to remove and re-add things in the Neumes customization.

### CODE REVIEW

**Review by bwbohl — APPROVED** [review]

**Review by fujinaga — APPROVED** [review]
I'm approving this at ODD.

**Comment by musicEnfanthen** [comment]
@yinanazhou Could you please check this PR against your use cases?

**Comment by ahankinson** [comment]
Added a comment on the associated issue to ask for confirmation that it fixes the problem.

**Comment by rettinghaus** [comment]
From [this comment](https://github.com/music-encoding/music-encoding/issues/968#issuecomment-1220767807) I conclude that this PR solves the problem with `<custos>` and `<clef>` in the neumes model. 
Before we continue, this should be merged. Waiting for a second approval by any @music-encoding/maintainers

**Comment by ahankinson** [comment]
I think probably @martha-thomae is the best one to review that

**Comment by martha-thomae** [comment]
I confirm the pull request works as intended. When using the generated schema for neumes: 
* `custos` is allowed outside syllable, but not inside
* `clef` is allowed both inside and out

Here is a screenshot corroborating this and a minimal MEI file you can use for testing these and other changes desired for the Neumes module (which are summarized in [this comment](https://github.com/music-encoding/music-encoding/issues/968#issuecomment-1146583421)).

klaus_customization_test

[testing_changes_to_neumes_module.xml.zip](https://github.com/music-encoding/music-encoding/files/9593014/testing_changes_to_neumes_module.xml.zip)

**Review by martha-thomae — APPROVED** [review]

**Pull request merged**
